lib/ddtrace/contrib/elasticsearch/patcher.rb

```diff
@@ -13,7 +13,7 @@ module Patcher
         # patch applies our patch if needed
         def patch
           if !@patched && (defined?(::Elasticsearch::Transport::VERSION) && \
-                           Gem::Version.new(::Elasticsearch::Transport::VERSION) >= Gem::Version.new('1.0.0'))
+                           Gem::Version.new(::Elasticsearch::Transport::VERSION) >= Gem::Version.new('1.1.1'))
```

Comment:
is this the minimum version we're going to support? was the previous ``1.0.0`` wrong?